Entry 2 -- November 8.md

```diff
@@ -15,12 +17,21 @@ design process.
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+I feel like my API is rudimentary at best. I spent a lot of time this week writing and rewriting the API structure because I couldn't find something I was *too* happy with. Any suggestions in this department would be helpful. Also, ScalaIDE/Eclipse dislikes my program structure and is unable to recognize class names in the same package. I spent some time trying to fix this issue but I think I will have to go to office hours/grutoring to try and fix this one.
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+What do you think about the API?
```

Comment:
A more specific prompt could be helpful here. How should a critique partner answer this question? Are there some properties of a good API (either general properties, e.g., from the reading we did earlier this semester; or properties that are specific to your domain) that we should be on the lookout for?